ui/src/main/java/com/senlainc/warsaw/tyurin/action/craftsman/AddCraftsman.java

```diff
@@ -29,6 +29,8 @@ public void execute() {
         System.out.println("Enter craftsman surname ");
```

Comment:
At this point of development you should get rid of all scanners in your application. Replace user input with prepared const data